### DISCUSSION TITLE
Feature Request: Strikethrough button in toolbar

### DISCUSSION THREAD

**Discussion by rjonesy** [community/community#16] [Mobile] [answered]
This is semi-related to https://github.com/github/feedback/discussions/12 request for Blockquote button. 

**My request is to add a strikethrough button** to the very nice toolbar you folks have implemented. We use strikethrough often because the first post in an Issue or Discussion is kinda like the overview or progress tracking – so if we decide _not_ to do something we strikethrough it to keep the record.

**Comment by rnystrom** [comment]
[Linking to my comment](https://github.com/github/feedback/discussions/17#discussioncomment-109339) on checkbox controls.

Strikethrough feels like a good quick-win, we will just need to think about how this bar scales as we add more controls.

**Comment by rnystrom** [comment, marked as answer]
Landed today and will go out in an upcoming beta!
